tests/utils/dockercli/volumelifecycle.go

```diff
@@ -53,8 +66,70 @@ func KillDocker(ip string) ([]byte, error) {
 	return out, err
 }
 
+// GetVDVSPlugin - get vDVS plugin id
+func GetVDVSPlugin(ip string) (string, error) {
+	out, err := ssh.InvokeCommand(ip, dockercli.GetVDVSPlugin)
+	if misc.FormatOutput(out) == "" {
+		return "", fmt.Errorf("vDVS plugin unavailable")
+	}
+	return strings.Fields(misc.FormatOutput(out))[0], err
+}
+
+// StartVDVSPlugin - manually start vDVS plugin
+func StartVDVSPlugin(ip string) ([]byte, error) {
+	log.Printf("Manually starting vDVS plugin on VM [%s]\n", ip)
+	out, err := ssh.InvokeCommand(ip, dockercli.StartVDVSPlugin)
+	return out, err
+}
+
+// KillVDVSPlugin - kill vDVS plugin. It is restarted automatically
+func KillVDVSPlugin(ip, pluginID string) ([]byte, error) {
+	out, err := ssh.InvokeCommand(ip, dockercli.KillVDVSPlugin+pluginID)
+	time.Sleep(2 * time.Second)
+	return out, err
+}
+
+// RestartVDVSPlugin - kill vDVS plugin. It is restarted automatically
+// if autorestart failed, start it manually
+func RestartVDVSPlugin(ip string) ([]byte, error) {
+	log.Printf("Restarting vDVS plugin on VM [%s]\n", ip)
+
+	pluginID, err := GetVDVSPlugin(ip)
+	if err != nil {
+		log.Printf("vDVS plugin unavailable")
+		return nil, err
+	}
+
+	out, err := KillVDVSPlugin(ip, pluginID)
+	if err != nil {
+		log.Printf("Killing vDVS plugin failed")
+		return out, err
+	}
+
+	pluginID, err = GetVDVSPlugin(ip)
```

Comment:
shouldn't we look for `enable/disable` status? we can say plugin is auto restarted incase of enable status isn't it?